libs/storage/tests/gtest/file_object/file_object_tests.cpp

```diff
@@ -195,7 +196,9 @@ TEST_F(FileObjectTests, ResizeAndWriteFiles)
 
   for (std::size_t i = 0; i < 10; ++i)
   {
-    std::random_shuffle(consistency_check_.begin(), consistency_check_.end());
+    std::random_device rd;
```

Comment:
I suspect that constructing this is expensive and not something that you want to do inside the loop

Reply:
I agree but it's a test, and presumably it's what was happening already inside random_shuffle?

Reply:
anyway, fixed